modules/nw-scaling-ingress-controller.adoc

```diff
@@ -0,0 +1,93 @@
+// Module filename: nw-scaling-ingress-controller.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-ingress-controller-configuration-{context}"]
+= Scaling an ingress controller
+
+An ingress controller can be manually scaled up or down based on
+requirements such as the need to increase throughput. `oc`
+commands are used to scale the `IngressController` resource.
+
+.Procedure
+. View the current number of replicas of the ingress controller resource.
++
+----
+$ name="foo"  <1>
+$ oc get ingresscontroller/$name -n openshift-ingress-operator -o yaml
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  creationTimestamp: 2019-04-09T16:11:52Z
+  finalizers:
+  - ingress.openshift.io/ingress-controller
+  generation: 2
+  name: $name
+  namespace: openshift-ingress-operator
+  resourceVersion: "1187245"
+  selfLink: /apis/operator.openshift.io/v1/namespaces/openshift-ingress-operator/ingresscontrollers/$name
+  uid: 303a2234-5ae2-11e9-a3f2-0adb90d769be
+spec:
+  replicas: 2
+status:
+  availableReplicas: 2
+  conditions:
+  - lastTransitionTime: 2019-04-09T16:12:27Z
+    status: "True"
+    type: Available
+  domain: apps.example.com
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+  selector: ingress.operator.openshift.io/ingress-controller-deployment=$name
+----
+<1> Replace "foo" with the name of the ingress controller you would like to scale.
+
+The `replicas` field represents the desired number of pods, while `availableReplicas`
+represents the number of a running pods. If an ingress controller is created without
+specifying `replicas`, the field will not appear in the output and a default value
+of `2` will be used.
```

Comment:
the description of availableReplicas is not quite precise... from the API docs:

> availableReplicas is number of observed available replicas according to the ingress controller deployment.

Reply:
I've simplified the doc and removed this content. IMO we can address the difference between the 2 fields in a separate doc and keep this focused on the tactical steps. Let me know if you disagree and I can reincorporate the content.